src/Admin/PageAdmin.php

```diff
@@ -167,6 +154,19 @@ public function setCacheManager(CacheManagerInterface $cacheManager)
         $this->cacheManager = $cacheManager;
     }
 
+    protected function configureBatchActions($actions): array
+    {
+        $actions = parent::configureBatchActions($actions);
+
+        $actions['snapshot'] = [
+            'label' => 'create_snapshot',
+            'translation_domain' => $this->getTranslationDomain(),
```

Comment:
isn't that unnecessary? the translation domain defaults to the Admins Translation domain anyway

Reply:
I just changed getBatchActions to configureBatchActions without changing the code inside.